### PR TITLE
Add next Java LTS to the WIP section of the contribution guide

### PR DESCRIPTION
**Pull request by iemejia**

### CODE REVIEW

**Comment by iemejia** [comment]
R: @kennknowles

**Comment by kennknowles** [comment]
Awesome.

**Comment by kennknowles** [comment]
@asfgit merge

File: src/contribute/index.md [hunk 1]
```diff
@@ -184,6 +184,13 @@ Work is in progress to add Python 3 support to Beam.  Current goal is to make Be
 Contributions are welcome! If you are interested to help, you can select a subpackage to port and assign yourself the corresponding issue. Comment on the issue if you cannot assign it yourself.
 When submitting a new PR, please tag [@RobbeSneyders](https://github.com/robbesneyders), [@aaltay](https://github.com/aaltay), and [@tvalentyn](https://github.com/tvalentyn).
 
+### Next Java LTS version support (Java 11 / 18.9)
+
+Work in progress to support the next LTS release of Java is in progress. For more details about the scope and info on the various tasks please see the JIRA ticket.
```
Comment (pabloem) [inline src/contribute/index.md]: For some reason this PR has not been merged, so I take my window of opportunity to mention:
This line says `"in progress"` twice. Maybe cut one out? : )

**Comment by iemejia** [comment]
Thanks good catch @pabloem. Fixed now.

**Comment by iemejia** [comment]
@asfgit merge

**Pull request closed without merging**